config/labels.json

```diff
@@ -0,0 +1,215 @@
+[
+  {
+    "name": "type: bug",
+    "aliases": [
+      "bug"
+    ],
+    "description": "Something isn't working",
+    "color": "FEF2C0"
+  },
+  {
+    "name": "type: enhancement",
```

Comment:
I suggest we split enhancements and features. For me a feature is something new, something added in terms of functionality that did not exist before, while enhancement is improvement of something that already exist and is available but could be made better in some sense.

I get that adding a new feature is at the same time improvement/enhancement of the product in a general sense, so just a suggestion.

Reply:
fixed in c06413c